website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md

```diff
@@ -0,0 +1,114 @@
+---
+title: An update on the New Architecture Rollout
+authors: [cortinico]
+tags: [announcement]
+---
+
+# An update on the New Architecture Rollout
+
+Hi everyone,
+[As previously announced](/blog/2022/01/21/react-native-h2-2021-recap#the-new-architecture-rollout-and-releases):
+
+> 2022 is going to be the year of the New Architecture in open source
+
+If you still haven’t had the time to look into the New React Native Architecture (the Fabric Renderer and the TurboModule system), there is no better time to do it **than now**!
+
+We would like to share with the community some initiatives and material we prepared to make sure everyone is onboard on this endeavor.
+
+<!--truncate-->
+
+### The Working Group
+
+Recently, we launched the [React Native New Architecture Working Group](https://github.com/reactwg/react-native-new-architecture) on GitHub, a _discussion only_ repository to coordinate and support the rollout of the New Architecture across the ecosystem.
+
+We envision this working group as a space where the community can **meet**, share **ideas**, and **discuss** challenges during the adoption of the New Architecture. Moreover, we're going to use this working group to **share** information and updates with the wider community for the sake of transparency.
+
+To keep the discussion focused, we decided to have this working group **open to read** publicly and **restricted to write** only for approved users.
+
+If you wish to join the conversation, you can [fill in this form](https://forms.gle/8emgdwFZXuzEpyyn9) to either **apply or nominate** someone that you think would be a valuable addition to the discussion.
+
+**Everyone is welcome** to apply to join the conversation.
+
+As every discussion forum, we would like to stress once more the importance of being **respectful** and welcoming towards others’ opinions. Please take the chance to read our [**code of conduct**](https://github.com/reactwg/react-native-new-architecture/blob/main/CODE_OF_CONDUCT.md) if you haven’t yet done it.
+
+### The Migration Guide
+
+After several rounds of review & feedback, we finally merged **the Migration Guide** (f.k.a. _the Playbook_). You can find it [on the website in the Guides section.](/docs/next/new-architecture-intro)
+
+This Migration Guide will show you **how to create a custom Fabric component or a TurboModule** with a step-by-step approach. The guide will also show you how to **adapt your existing app or library** to use the New Architecture.
+
+Moreover, we would like to remind you the brand-new [Architecture section](/docs/next/architecture-overview) of our website. There you can find several in-depth articles and explanation of the React Native internals. Specifically, [the Fabric section](/docs/next/fabric-renderer) can help you understand the rendering pipeline in the New Architecture world.
```

Comment:
```suggestion
Moreover, we would like to remind you the brand-new [Architecture section](/architecture/overview) of our website. There you can find several in-depth articles and explanation of the React Native internals. Specifically, [the Fabric section](/architecture/fabric-renderer) can help you understand the rendering pipeline in the New Architecture world.
```

Links have changed after latest conversion of the architecture section page to unversioned docs.